README.md

```diff
@@ -124,11 +130,12 @@ initialization scripts (and later daemons) to pick from.
 ovs-vsctl set Open_vSwitch . external_ids:k8s-api-server="0.0.0.0:8080"
 ```
 
-Install the following additional packages for python.
+Clone this repository and install the executables.
 
 ```
-pip install netaddr
-pip install eventlet
+git clone https://github.com/openvswitch/ovn-kubernetes
+cd ovn-kubernetes
+pip install -e `pwd`
```

Comment:
You can also just use "." instead of "`pwd`".

It might be worth explaining what this does.  Using "-e" means that any changes you make to the Python code are immediately reflected in the installed version, as what's installed just points back to your git tree for the actual code to run.